.github/workflows/docker-publish.yml

```diff
@@ -61,25 +61,28 @@ jobs:
       # https://github.com/docker/metadata-action
       - name: Extract metadata
         id: meta
-        uses: docker/metadata-action@96383f45573cb7f253c731d3b3ab81c87ef81934 # v5.0.0
+        uses: docker/metadata-action@902fa8ec7d6ecbf8d84d538b9b233a880e428804 # v5.7.0
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
             type=semver,pattern={{version}}
             type=sha
-            type=raw,value=latest
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
```

Comment:
Why not just
```
GitHub.ref == 'refs/head/main'
```

Reply:
Good point! I forgot about changing that from the docs example.